backend/scraper/scrape_walmart.py

```diff
@@ -19,20 +20,31 @@ def __init__(self):
             chrome_options = Options()
             chrome_options.add_argument('--headless')
             try:
-                driver = r'./chromedriver.exe'
-                self.driver = webdriver.Chrome(driver, options=chrome_options)
+                self.driver = webdriver.Chrome(ChromeDriverManager().install())
+                self.driver.get("https://www.walmart.com/")
+                #self.driver.get = webdriver.Chrome(driver, options=chrome_options)
+                # Get cookies for request session - needed for getting each product
+                cookies = self.driver.get_cookies()
+                self.driver.quit()

```

Comment:
Okay, so I found the bug: The reason we're getting the error is that the Selenium Driver quits Chrome, and then tries to make requests to the driver. You'll need to delete this line here and also revert line 46 and 62 to use `self.driver.get`.

The difference between the Walmart scraper and HEB scraper is that the Walmart scraper needs us to search for the items in an active Chrome window. Once we have the item list, we can use Python Requests to get information about each individual item. However, in the HEB scraper, we can use Python Requests right after we get a fresh set of cookies, which is why we use `self.session` instead of `self.driver`